src/test/java/com/esri/core/geometry/TestOGC.java

```diff
@@ -955,5 +954,86 @@ public void testEmptyBoundary() throws Exception {
 		}
 	}
 	
+	@Test
+	public void testUnionPointWithEmptyLineString() {
+		assertUnion("POINT (1 2)", "LINESTRING EMPTY", "POINT (1 2)");
+	}
+
+	@Test
+	public void testUnionPointWithLinestring() {
```

Comment:
let's remove this test; it is already included in `TestOGCGeometryCollection#testUnionPoint`